<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractionsTodo</class>
 <widget class="QWidget" name="InteractionsTodo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>724</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ajout_Interaction_Todo</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_10">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <widget class="QLabel" name="Nom_3">
         <property name="text">
          <string>Nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="NomE_3"/>
       </item>
       <item>
        <widget class="QLabel" name="Prenom_3">
         <property name="text">
          <string>Pr√©nom</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="PrenomE_3"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextBrowser" name="Affichage_3"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_8"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <item>
        <widget class="QLabel" name="Interactions">
         <property name="text">
          <string>Interactions</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="InteractionsE"/>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_11">
         <item>
          <widget class="QLabel" name="Todo">
           <property name="text">
            <string>Todo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="Valide">
           <property name="minimumSize">
            <size>
             <width>206</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>20</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="TodoE"/>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="DateTodo"/>
       </item>
       <item>
        <widget class="QPushButton" name="AjoutTodo">
         <property name="text">
          <string>AjoutTodo</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QPushButton" name="Interact">
       <property name="text">
        <string>Ajout Interactions</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Annuler_3">
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
